### PR TITLE
Add a basic bootstrap spinner to screens that reload

### LINKED ISSUE #15: Add a spinner?

### PR DESCRIPTION
**Pull request by nonprofittechy**
Fix #15

### CODE REVIEW

File: docassemble/ALKilnInThePlayground/data/questions/run_alkiln_tests.yml [hunk 1]
```diff
@@ -324,6 +324,12 @@ event: wait_for_install
 question: |
   One sec, installing ALKiln v${version_to_install}
 subquestion: |
+  <div class="d-flex justify-content-center">
+  <div class="spinner-border" style="width: 3rem; height: 3rem;" role="status">
+    <span class="visually-hidden">Loading...</span>
```
Comment (nonprofittechy) [inline docassemble/ALKilnInThePlayground/data/questions/run_alkiln_tests.yml:329]:
```suggestion
    <span class="visually-hidden">Installing...</span>
```

File: docassemble/ALKilnInThePlayground/data/questions/run_alkiln_tests.yml [hunk 2]
```diff
@@ -487,6 +493,12 @@ reload: True
 question: |
   Hang tight, ALKiln is running the tests.
 subquestion: |
+  <div class="d-flex justify-content-center">
+  <div class="spinner-border" style="width: 3rem; height: 3rem;" role="status">
+    <span class="visually-hidden">Loading...</span>
```
Comment (nonprofittechy) [inline docassemble/ALKilnInThePlayground/data/questions/run_alkiln_tests.yml:498]:
```suggestion
    <span class="visually-hidden">Running tests...</span>
```

File: docassemble/__init__.py [hunk 3]
```diff
@@ -1,5 +1,2 @@
-try:
-    __import__('pkg_resources').declare_namespace(__name__)
-except ImportError:
-    __path__ = __import__('pkgutil').extend_path(__path__, __name__)
+__import__('pkg_resources').declare_namespace(__name__)
```
Comment (plocket) [inline docassemble/__init__.py:1]: I'm ready to merge this other than this change which popped itself in here for some reason. Does anyone know why it did that? Should this be changed back?

Comment (nonprofittechy) [reply docassemble/__init__.py:1]: Seems like an upstream Docassemble change which is likely fine, unless we had to intentionally change from the Docassemble defaults in an earlier commit

Comment (nonprofittechy) [reply docassemble/__init__.py:1]: I'm guessing the original behavior was legacy for python 2.7.x

**Review by plocket — APPROVED** [review]
Thanks for the contribution! We've wanted the spinner from the start, and it's good to have it in there now.

**Pull request merged**